multiversx_sdk/network_providers/transaction_awaiter_test.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+
+from multiversx_sdk.core.address import Address
+from multiversx_sdk.core.transaction import Transaction
+from multiversx_sdk.core.transaction_computer import TransactionComputer
+from multiversx_sdk.network_providers.proxy_network_provider import \
+    ProxyNetworkProvider
+from multiversx_sdk.network_providers.transaction_awaiter import \
+    TransactionAwaiter
+from multiversx_sdk.network_providers.transaction_status import \
+    TransactionStatus
+from multiversx_sdk.network_providers.transactions import TransactionOnNetwork
+from multiversx_sdk.testutils.mock_network_provider import (
+    MockNetworkProvider, TimelinePointMarkCompleted, TimelinePointWait)
+from multiversx_sdk.testutils.wallets import load_wallets
+
+
+class ProxyWrapper:
+    def __init__(self, proxy: ProxyNetworkProvider) -> None:
+        self.proxy = proxy
+
+    def get_nonce(self, address: Address) -> int:
+        return self.proxy.get_account(address).nonce
+
+    def get_transaction(self, tx_hash: str) -> TransactionOnNetwork:
+        return self.proxy.get_transaction(tx_hash, True)
```

Comment:
Maybe can we write the tests without defining `get_nonce` and `send_transaction` in `ProxyWrapper`? That is, only have `get_transaction` here?

Otherwise, people might think that it's necessary to define such a wrapper (with 3 functions) if you need to create a `TransactionAwaiter` using a `ProxyNetworkProvider`.

Reply:
Indeed. Done!